src/js/index.js

```diff
@@ -1,2 +1,41 @@
 import 'bootstrap/dist/css/bootstrap';
 import $ from 'jquery';
+
+const url = 'https://api.coindesk.com/v1/bpi/currentprice.json';
+fetch(url)
+  .then(data => data.json())
+  .then(res => {
+    for (let key in res) {
+      let detail = res[key];
+      switch (key) {
+        case 'chartName':
+          $('#main h1').html(`${detail}`);
+          break;
+        case 'time':
+          $('div.updatedTime').html(`Updated: ${detail.updated}`);
+          break;
+        case 'disclaimer':
+          $('div.description').html(`${detail}`);
+          break;
+        case 'bpi':
+          for (let k in res[key]) {
+            $('#main table')
+              .add(
+                `
+                        <tr>
+                        <td>${detail[k].code}</td>
+                        <td>${detail[k].description}</td>
+                        <td>${detail[k].rate}</td>
+                        <td>${detail[k].symbol}</td>
+                        </tr>
+                    `
+              )
+              .appendTo('#main table');
+          }
+          break;
```

Comment:
Interesting way of building the data with DOM.

But it is not very readable code to me when you have a big switch statement within a for in loop.

Shall we try the native Array reduce method if possible?